template/pagetype-doc.html.jinja

```diff
@@ -29,6 +29,61 @@
     <div class="content">
         {{ content }}
     </div>
+    <div class="discord-popup">
+      <div class="popup-window" id="myPopup">
+        <button type="button" class="close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        <p class="popup-text">Have questions?</p>
+        <a href="http://xrpldevs.com" class="btn btn-primary btn-arrow">
+          <span class="discord-logo"></span>
+          <span>Join the XRPL Discord</span>
+        </a>
+        <label class="show-again">
+        <input type="checkbox" class="show-again-checkbox">
+        <span class="show-again-checkmark"></span>
+          <label for="dont-show-again" class="show-again-message">Don't show this again</label>
+        </label>
+      </div>
+    </div>
+
+    <script type="text/javascript">
```

Comment:
Can we move the scripts out to a separate `.js` file (or add it to a shared one) instead of inlining them? I don't think it's good practice to have JS code scattered into template files ad-hoc.